app/lib/searching.rb

```diff
@@ -0,0 +1,62 @@
+class Searching
+  require 'gds_api/rummager'
+  attr_reader :params
+  def initialize(params)
+    @params = params
+  end
+
+  def count
+    return 10 if params["count"].blank?
+    return 1000 if params["count"].to_i > 1000
+    params["count"]
+  end
+
+  def call
+    terms = params["search_term"].split.join("+")
+    rummager = GdsApi::Rummager.new(Plek.new.find('rummager'))
+    fields = %w(
+      title link public_timestamp description format
+      )
+    findings_new_left = rummager.search(q: terms, fields: fields, count: count.to_s,
```

Comment:
I don't know if the linter picks this up, but a common way of indenting function calls that don't fit on one line (in our codebases) is like

```ruby
rummager.search(
  q: terms,
  fields: fields,
  count: count.to_s,
  ab_tests: "#{params['which_test']}:A",
  c: Time.now.getutc.to_s
)
```

it just makes it easier to visually scan the parameter list when they're all aligned 🙂